packages/frontend/styles/ddsIcons.ts

```diff
@@ -0,0 +1,470 @@
+    'docs-copy-alt',
+    'doc-search',
+    'doc-secure',
+    'doc-settings',
+    'doc-support',
+    'download',
+    'eye-view-off',
+    'filter-add',
+    'filter-clear',
+    'filter-move',
+    'filter-remove',
+    'folder-arrow-up',
+    'folder-media',
+    'gear-arrow',
+    'gear-wrench',
+    'graphics-card',
+    'hand-touch-screen',
+    'hd-generic',
+    'headphones',
+    'headset',
+    'heat',
+    'help-sqr',
+    'host-server',
+    'host-server-hex',
+    'host-server-rack',
+    'inbox-arrow-down',
+    'infrastructure',
+    'keypad',
+    'lcd',
+    'location-play',
+    'log-out',
+    'leaf',
+    'manage-info',
+    'management-software',
+    'memory',
+    'memory-card-reader',
+    'memory-sd-card',
+    'microchip',
+    'microphone-on',
+    'migrate',
+    'minimize',
+    '2-in-1',
+    'pre-seed',
+    'receipt',
+    'brand-xps',
+    'data-protection',
+    'hyper-converged-infrastructure',
+    'unpin',
+    'network-connected',
+    'network-horiz',
+    'network-share',
+    'network-vert-disable',
+    'partnership',
+    'password',
+    'pause-sqr',
+    'pencil-remove',
+    'popout-double',
+    'popout-share',
+    'popout-window-arrow',
+    'pop-up-arrow-corner',
+    'port-double',
+    'power-off',
+    'power-on',
+    'protractor',
+    'pulse',
+    'puzzle',
+    'qos',
+    'rack',
+    'rss',
+    'search-arrow-right',
+    'server-rack-arrow-right',
+    'shipping',
+    'skip-forward',
+    'skip-to-end',
+    'skip-to-front',
+    'slider-horiz',
+    'smart-select',
+    'social-networking',
+    'start-up',
+    'suspend',
+    'sync-vert',
+    'task-check',
+    'task-check-progress',
+    'usb',
+    'user-about',
+    'user-add',
+    'user-check',
+    'user-cir',
+    'user-cir-add',
+    'user-group-disable',
+    'user-groups-disable',
+    'user-lock',
+    'user-remove',
+    'user-search',
+    'user-unlock',
+    'video-card',
+    'view-detailed',
+    'web-camera',
+    'web-camera-hd',
+    'wifi-simple',
+    'windows-support',
+    'wrench-cir',
+    'wrench-double',
+    'package-closed',
+    'package-open',
+    'pie-chart',
+    'pie-chart-solid',
+    'power-supply-cord',
+    'power-supply-double',
+    'processor',
+    'productivity',
+    'removable-storage-usb',
+    'server-rack-search',
+    'server-rack-support',
+    'shield-protect',
+    'sort-za',
+    'speaker-double',
+    'stack',
+    'toolbox',
+    'touch-pad',
+    'translate',
+    'user-cir-remove',
+    'user-contact',
+    'user-disable',
+    'user-group-chat',
+    'view-list',
+    'view-row',
+    'view-table',
+    'virtualization',
+    'wrench-manage',
+    'zoom-in',
+    'zoom-out'
+];
+
+const iconClassName = (iconSuffix: string) => `icon-button dds__icon dds__icon--${iconSuffix}`;
+
+const dashToCamelCase = (dashCase: string) => dashCase.replace(/-([a-z])/g, g => g[1].toUpperCase());
+
+export const icons: Record<typeof iconsSuffix[number], string> = iconsSuffix.reduce(
```

Comment:
As `typeof iconsSuffix[number]` is now returning `string`, we can simplify code, by using it directly 💪 

```suggestion
export const icons: Record<string, string> = iconsSuffix.reduce(
```

I suspected that the thought behind the implementation was to list all available icon names, which could be accessed from `icons` object, but to make this solution work, some structural changes would need to happen